block/testdata/deserialize.json

```diff
@@ -0,0 +1,23 @@
+[
```

Comment:
What's the origin of these fixtures? Did you already check if we can reuse some from the elements repository? It's likey that they already have fixtures for these operations and would be cool to share the same test vectors.

Reply:
I used some of fixtures from [Rust repo](https://github.com/ElementsProject/rust-elements/blob/0d67c57afa1137ab27861bb8c2190413929d4301/src/block.rs#L383) also added one from local environment.

Reply:
cool! :) would be nice to add this reference somewhere in the code, like in one of the test file.

Reply:
Done